documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -2072,9 +2072,34 @@ After a source record is deleted, a tombstone event (the default behavior) enabl
 
 |[[oracle-property-message-key-columns]]<<oracle-property-message-key-columns, `+message.key.columns+`>>
 |No default
-| A list of regular expressions, separated by semi-colons, that match the fully-qualified tables and columns to map a primary key. +
-Each item (regular expression) must match the `<fully-qualified table>:<a comma-separated list of columns>` representing the custom key. +
-Define fully-qualified tables by using the following format: `_<pdbName>_._<schemaName>_._<tableName>_`.
+|A list of expressions that specify the columns that the connector uses to form custom message keys for change event records that it publishes to the Kafka topics for specified tables.
+
+By default, {prodname} uses the primary key column of a table as the message key for records that it emits.
+In place of the default, or to specify a key for tables that lack a primary key, you can configure custom message keys based on one or more columns. +
+ +
+To establish a custom message key for a table, list the table, followed by the columns to use as the message key.
+Each list entry takes the following format: +
+ +
+`_<fully-qualified_tableName>_:_<keyColumn>_,_<keyColumn>_` +
+ +
+To base a table key on multiple column names, insert commas between the column names.
+
+Each fully-qualified table name is a regular expression in the following format: +
+ +
+`_<pdbName>_._<tableName>_` +
```

Comment:
@Naros could you check this out: is the PDB name actually part of this? Or is it just schema name + table name?

Reply:
@gunnarmorling Looking at the integration tests it seems the user can supply the format as either `<pdbName>.<schemaName>.<tableName>` or `<schemaName>.<tableName>` and it works.

@roldanbob I don't agree, if you look at the diff the original published document uses the format:
`<pdbName>.<schemaName>.<tableName>`
however the changed view of the oracle.adoc now says:
`<pdbName>.<tableName>`

I would say we should prefer the second flavor for consistency with most other connectors and this line actually be 

```suggestion
`_<schemaName>_._<tableName>_` +
```

Reply:
@Naros @gunnarmorling  Oh, shoot! Sorry, you're right, of course. I had created a new boilerplate description and had noted that there were exceptions for Oracle and MySQL from the `schemaName.table.Name` value that most of the connectors use, but in my notes I only focused on the `pdbName` part. 

So, wait, you're saying that besides just making the correction, I should change the value to `schemaName.tableName`, and not mention `pdbName` at all? If that's the case, then my error was useful in bringing this to your attention. ;)

I'll fix and push the change.

Reply:
>  should change the value to schemaName.tableName, and not mention pdbName at all?

Correct @roldanbob.  This way it's consistent with connectors like db2 / sqlserver.

Reply:
Ok, got it. Thank you for the input.

Reply:
BTW, in a search of the document I found other references to `pdbName.schemaName.tableName` in the description for `column.mask.hash._<hashAlgorithm>_.with.salt._<salt>_` on L2001, `column.truncate.to._length_.chars` on L2107, and `column.mask.with._length_.chars` on L2116. 

Should I likewise change those?

Reply:
Good question; I'd be inclined to say that the PDB name shouldn't be required, as currently the connector will always work within the context of a single PDB (similar to as we for instance don't require the database name for similar options for the Postgres connector, as it is connected to a single DB). This _may_ change in the future (see work happening by @morozov for DBZ-2975 for SQL Server), but we're not there yet. But it's why I wouldn't actively unsupport the PDB name being given in these options either, I'd only not document it for now.

Reply:
Hi @roldanbob, yes this works in an identical way as the `message.key.columns` where the connector supports both the syntax with and without the PDB database name.  So +1 to @gunnarmorling suggestion that let's just document it without the `pdbName` portion and we'll add that back in when we explore multi-tenancy support with the Oracle connector.

Reply:
> let's just document it without the `pdbName` portion and we'll add that back in when we explore multi-tenancy support with the Oracle connector.

Ok, cool. Thanks. I'll make the changes now.

Reply:
For reference, here's a potentially related PR: https://github.com/debezium/debezium/pull/2755. Not sure if it affects this exact feature though (at least not according to the tests).